bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/ClientLibrariesImpl.java

```diff
@@ -81,11 +82,11 @@ public class ClientLibrariesImpl implements ClientLibraries {
     @Named(OPTION_RESOURCE_TYPES)
     Object resourceTypes;
 
-    @Inject
+    @ValueMapValue
```

Comment:
I think using the element "name" instead of annotation @Named is even more concise.